tools/serverpod_cli/lib/src/analyzer/models/entity_dependency_resolver.dart

```diff
@@ -30,6 +33,46 @@ class ModelDependencyResolver {
     });
   }
 
+  static void _resolveInheritance(
+    ClassDefinition classDefinition,
+    List<SerializableModelDefinition> modelDefinitions,
+  ) {
+    if (classDefinition.extendsClass != null) {
+      try {
+        var baseClass = modelDefinitions
+            .whereType<ClassDefinition>()
+            .firstWhere((element) =>
```

Comment:
Question: The `try/catch` statement here is only for the `firstWhere` statement right?
I would recommend either isolating that specific filtering or finding a non throwing way to find out if an elelement exists. For example the use of `firstWhereOrNull`.